sdks/python/apache_beam/io/gcp/bigquery_file_loads.py

```diff
@@ -550,6 +562,25 @@ def verify(self):
                        'loaded into BigQuery. Please provide a GCS bucket, or '
                        'pass method="STREAMING_INSERTS" to WriteToBigQuery.'
                        % self._custom_gcs_temp_location.get())
+    if self.is_streaming_pipeline and not self.triggering_frequency:
+      raise ValueError('triggering_frequency must be specified to use file'
+                       'loads in streaming')
+    elif not self.is_streaming_pipeline and self.triggering_frequency:
+      raise ValueError('triggering_frequency can only be used with file'
+                       'loads in streaming')
+
+  def _window_fn(self):
+    if self.is_streaming_pipeline:
+      return beam.WindowInto(beam.window.GlobalWindows(),
+                             trigger=trigger.Repeatedly(
+                                 trigger.AfterAny(
+                                     trigger.AfterProcessingTime(
+                                         self.triggering_frequency),
+                                     trigger.AfterCount(
+                                         _FILE_TRIGGERING_RECORD_COUNT))),
```

Comment:
I agree. With the current trigger, for large no. of files, the quota will be exceeded since multiple load jobs will be triggered. 
The Java implementation applies this trigger, writes the files and then triggers load jobs only based on the triggering frequency[1]. I now understand why that second trigger exists. I'll add it to this.

[1] https://github.com/apache/beam/blob/master/sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BatchLoads.java#L272-L281

Reply:
Right, that trigger checks that there's at least one element, and then it waits the triggering frequency.

Reply:
Adding it..

Reply:
Added